lib/config/executor/container_vm.go

```diff
@@ -108,7 +123,7 @@ type ContainerVM struct {
 // in that there is no process inherently associated - this is closer to a ThreadPool than a Thread and
 // is the owner of the shared filesystem environment. This is the guest visible complement to ContainerVM.
 type ExecutorConfig struct {
-	Common `vic:"0.1" scope:"read-only" key:"common"`
```

Comment:
It's still.. Idk, if there's no other way to do it, so be it, but it just kills me to have such similar structs with such similar names and even the same comments right next to each other. I mean I know we don't have classic inheritance but this just jumps out at me as begging for some sort of polymorphism or encapsulation to make the naming more succinct/clearer and/or to reuse code and not have to copy/paste..

But maybe it's not possible here, I don't know, it just seems like there should be a cleaner way. If not, I'll live.